content/departments/technical-success/support/process/product-gap-process.md

```diff
@@ -0,0 +1,11 @@
+# Product gaps
+
+When we work through support tickets, we often come across product gaps. Product gaps can be new feature requests or enhancements to our existing features, which might be viable for inclusion in our future roadmap. It's important to distinguish that a product gap isn't a bug or a request that strays too far from the core functionality and intended use of Sourcegraph. For more insight on product gaps, please consult the following section of our handbook: [Surfacing Product Feedback](../../../product/process/feedback/surfacing_product_feedback.md#product-gaps).
+
+Product gaps are submitted by the customer’s TA or Scaled Success for accounts without a TA. Thanks to the Vitally integration, you'll find the TA listed in the ZenDesk ticket under the Apps section. If it's coming up empty, head over to Vitally directly and use the search to find the account.
+
+**For customers with a TA**: Head to the #discuss-technical-advisors channel and follow the Product Gap Request workflow. Keep the ticket open for a few days in case the TA or customer has more to say.
+
+**For customers without a TA**: Head to the #discuss-scaled-success channel and follow the Product Gap Request workflow. Leave the ticket open for a few days, allowing the team or customer to follow up.
```

Comment:
These should also be marked as `Scaled Success` under `Support Level` on Vitally. If that helps with identifying say a named TA account that simply doesn't have a named TA yet (i.e. someone left and the account hasn't been assigned) vs a true Scaled Success account.